### PR TITLE
nao_lola: 0.0.2-1 in 'galactic/distribution.yaml' [bloom]

### PR DESCRIPTION
**Pull request by ijnek**
Increasing version of package(s) in repository `nao_lola` to `0.0.2-1`:

- upstream repository: https://github.com/ijnek/nao_lola.git
- release repository: https://github.com/ijnek/nao_lola-release.git
- distro file: `galactic/distribution.yaml`
- bloom version: `0.10.7`
- previous version for package: `null`

### CODE REVIEW

**Review by ahcorde — APPROVED** [review]

**Comment by ahcorde** [comment]
there are some conflicts, do you mind to solve it ?

File: galactic/distribution.yaml [hunk 1]
```diff
@@ -1562,6 +1562,19 @@ repositories:
...
+      url: https://github.com/ijnek/nao_lola.git
+      version: main
+    release:
+      tags:
+        release: release/galactic/{package}/{version}
+      url: https://github.com/ijnek/nao_lola-release.git
+      version: 0.0.2-1
+    source:
+      type: git
+      url: https://github.com/ijnek/nao_lola.git
```
Comment (tfoote) [inline galactic/distribution.yaml:1577]: It looks like your merge fixup missed lost the branch name:

```suggestion
      url: https://github.com/ijnek/nao_lola.git
      version: main
```

Comment (ijnek) [reply galactic/distribution.yaml:1577]: Ah, thanks!

**Comment by ijnek** [comment]
@tfoote 
Would you be able to merge? Thanks.

**Review by cottsay — CHANGES_REQUESTED** [review]
Looks like the `status` value from the original commit was also lost in the merge. I'm not sure if that is formally required, but it should probably be added back prior to merging.

**Comment by ijnek** [comment]
@cottsay 
Thanks for picking that up too, I should've double checked... I think everything is correct now.

**Review by cottsay — APPROVED** [review]
Thanks!

**Pull request merged**